src/components/sharedPageComponents/footer.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import facebook from '../../assets/Images/facebook_icon.svg';
+import twitter from '../../assets/Images/twitter_icon.svg';
+import youtube from '../../assets/Images/youtube_icon.svg';
+import instagram from '../../assets/Images/instagram-icon.svg';
+
+export default function Footer() {
+  return (
+    <>
+      <footer>
+        <div className="footer-banner">
+          <div className="copyright">
+            <p>&copy;codeit - 2023</p>
+          </div>
+          <div className="description">
+            <a href="/privacy">Privacy Policy</a>
+            <a href="/faq">FAQ</a>
+          </div>
+          <div className="footer-icon">
+            <a href="https://www.facebook.com" target="_blank" rel="noreferrer">
+              <img src={facebook} alt="facebook icon" width="20" height="20" />
+            </a>
+            <a href="https://twitter.com" target="_blank" rel="noreferrer">
+              <img src={twitter} alt="twitter icon" width="20" height="20" />
+            </a>
+            <a href="https://youtube.com" target="_blank" rel="noreferrer">
+              <img src={youtube} alt="youtube icon" width="20" height="20" />
+            </a>
+            <a
+              href="https://www.instagram.com"
+              target="_blank"
+              rel="noreferrer"
+            >
+              <img
+                src={instagram}
+                alt="instagram icon"
+                width="20"
+                height="20"
+              />
+            </a>
```

Comment:
```suggestion

            {SNS_LIST.map(({ name, href, src }) => {
              return (
                <a key={name} href={href} target="_blank" rel="noreferrer">
                  <img src={src} alt={`${name} icon`} width="20" height="20" />
                </a>
              );
            })}
```

```js

const SNS_LIST = [
  { name: 'facebook', href: 'https://www.facebook.com', src: facebook },
  { name: 'twitter', href: 'https://twitter.com', src: twitter },
  { name: 'youtube', href: 'https://youtube.com', src: youtube },
  { name: 'instagram', href: 'https://www.instagram.com', src: instagram },
];
```

이런식으로 배열을 구성한 뒤 map 돌리는 로직으로 구현하게 되면 추후 SNS 아이콘이 추가/수정/삭제될때 대응하기 편해집니다.